app/[locale]/(dashboard)/sensors/[id]/edit/page.jsx

```diff
@@ -0,0 +1,20 @@
+import { getServerSession } from 'next-auth/next'
+import { authOptions } from '@/lib/auth'
+import { Page } from '@/components/Page/Page'
+import { PageHeader } from '@/components/Page/PageHeader'
+
+async function Home({ params }) {
```

Comment:
By convention, I think we can call the component by name related with the page route like `SensorEdit`